index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Twitter clone</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link 
+    rel="stylesheet" 
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" 
+    integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+    <!-- building side, it starts here -->
+    <div class="sidebar">
+        <i class="fab fa-twitter"></i>
+
+    <div class="sidebarOption active">
+        <span class="material-icons"> home </span>
+        <h2>Home</h2>
+        </div>
```

Comment:
Your indentation is a little off, I'm not sure where one div ends and another begins. This would be easier to maintain like this:


```html
    <div class="sidebar">
        <i class="fab fa-twitter"></i>

        <div class="sidebarOption active">
            <span class="material-icons"> home </span>
            <h2>Home</h2>
        </div>
```